Integrations/integration-ArcherRSA.yml

```diff
@@ -2081,6 +2081,7 @@ script:
 
         if (records && records.Contents && records.Contents.length > 0) {
             for (var i = 0; i < records.Contents.length; i++) {
+                details = "";
```

Comment:
@guykeller is this used for anything? If not, please remove

Reply:
This is what fixes the concatenation issue.
As you can see, the "details" variable is declared outside the scope of the for loop, thus isn't re-written for each record.
This fixes it.